cmd/dump.go

```diff
@@ -55,6 +55,12 @@ It can be used for backup and capture Gitea server image to send to maintainer`,
 	},
 }
 
+func fatal(format string, args ...interface{}) {
```

Comment:
is this really needed? Also why then in some places this function is used while in others not

Reply:
My intent was to give error feedback also to a console so users don't have dig through logs to find an issue. I guess this method is better than adding additional `fmt.Fprintf` everywhere before `log.Fatal`.
Also logging to stderr should help troubleshooting when `dump` is run from the cron.
What do you think?

Reply:
And yea, those other places was just me overlooked them. Good catch! :)